packages/nitro-protocol/contracts/ETHAssetHolder.sol

```diff
@@ -67,7 +67,7 @@ contract ETHAssetHolder is AssetHolder {
      * @param amount Quantity of wei to be transferred.
      */
     function _transferAsset(address payable destination, uint256 amount) internal override {
-        (bool success, ) = destination.call{value: amount}('');
-        require(success, 'Could not transfer asset');
+        destination.send(amount); // forwards a stipend of only 2300 gas, will not revert on failure
+        // We do not require success here, to block a griefing vector
```

Comment:
There's an issue with this approach in that an `AssetTransferred` event will be emitted regardless. So we _should_ inspect the `.send()` return value to properly conditionalise the event emission.

Reply:
Fixed in 7e276cc but requires breaking change (expect some tests to fail).